src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -40,31 +43,10 @@ public void setCountry(String country) {
         this.country = country;
     }
 
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
-        Manufacturer that = (Manufacturer) o;
-        return Objects.equals(id, that.id)
-                && Objects.equals(name, that.name)
-                && Objects.equals(country, that.country);
-    }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(id, name, country);
-    }
-
```

Comment:
why you deleted equels and hashcode methods? return them and add them to the Driver class